scripts/spack/packages/axom/package.py

```diff
@@ -290,25 +300,19 @@ def initconfig_hardware_entries(self):
             entries.append(cmake_cache_option("CMAKE_CUDA_SEPARABLE_COMPILATION", True))
 
             # CUDA_FLAGS
-            cudaflags = "-restrict --expt-extended-lambda "
+            cudaflags = "${CMAKE_CUDA_FLAGS} -restrict --expt-extended-lambda "
```

Comment:
The cached cmake package from Spack now defines `CMAKE_CUDA_FLAGS` and `CMAKE_CUDA_ARCHITECTURES`.